runtime/near-vm-runner/src/wasmer2_runner.rs

```diff
@@ -284,53 +272,70 @@ fn run_method(module: &Module, import: &ImportObject, method_name: &str) -> Resu
     Ok(())
 }
 
-pub(crate) fn compile_wasmer1_module(code: &[u8]) -> bool {
-    let store = default_wasmer1_store();
+pub(crate) fn compile_wasmer2_module(code: &[u8]) -> bool {
+    let store = default_wasmer2_store();
     Module::new(&store, code).is_ok()
 }
 
+#[derive(Hash, PartialEq, Debug)]
+#[allow(unused)]
+enum WasmerEngine {
+    Universal = 1,
+    StaticLib = 2,
+    DynamicLib = 3,
+}
+
+#[derive(Hash, PartialEq, Debug)]
+#[allow(unused)]
+enum WasmerCompiler {
+    Singlepass = 1,
+    Cranelift = 2,
+    Llvm = 3,
+}
+
```

Comment:
Could we add some comments on what these enums are?

Reply:
Sure, will fix that. Generally, just different ways to compile Wasm to native.